Lib/_pyrepl/fancycompleter.py

```diff
@@ -0,0 +1,205 @@
+#   Copyright 2010-2025 Antonio Cuni
+#                       Daniel Hahler
+#
+#                        All Rights Reserved
+"""
+Colorful TAB completion for Python prompt
+"""
+from _pyrepl import readline
+from _colorize import ANSIColors
+import rlcompleter
+import types
+from itertools import count
+
+
+class DefaultConfig:
+
+    consider_getitems = True
+    use_colors = 'auto'
+
+    color_by_type = {
+        types.BuiltinMethodType: ANSIColors.BOLD_TEAL,
+        types.MethodType: ANSIColors.BOLD_TEAL,
+        type((42).__add__): ANSIColors.BOLD_TEAL,
+        type(int.__add__): ANSIColors.BOLD_TEAL,
+        type(str.replace): ANSIColors.BOLD_TEAL,
```

Comment:
```suggestion
        types.BuiltinMethodType: ANSIColors.BOLD_CYAN,
        types.MethodType: ANSIColors.BOLD_CYAN,
        type((42).__add__): ANSIColors.BOLD_CYAN,
        type(int.__add__): ANSIColors.BOLD_CYAN,
        type(str.replace): ANSIColors.BOLD_CYAN,
```

Reply:
Hugo's suggestion & using types from `types`
```suggestion
        types.BuiltinMethodType: ANSIColors.BOLD_CYAN,
        types.MethodType: ANSIColors.BOLD_CYAN,
        types.MethodWrapperType: ANSIColors.BOLD_CYAN,
        types.WrapperDescriptorType: ANSIColors.BOLD_CYAN,
        types.MethodDescriptorType: ANSIColors.BOLD_CYAN,
```